src/main/java/com/base/site/controllers/FoodController.java

```diff
@@ -0,0 +1,147 @@
+package com.base.site.controllers;
+
+import com.base.site.models.Food;
+import com.base.site.repositories.FoodRepo;
+import com.base.site.security.FoodService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+
+import javax.servlet.http.HttpSession;
+import javax.validation.Valid;
+import java.util.logging.Logger;
+
+@Controller
+public class FoodController {
+    Logger log = Logger.getLogger(FoodController.class.getName());
+
+    @Autowired
+    FoodRepo foodRepo;
+    //@Qualifier("FoodService")
+    @Autowired
+    FoodService foodService;
+
+
+    @GetMapping("/food")
+    public String food( Model model) {
+        model.addAttribute("foodlist", foodService.getAllFood());
+        log.info("  get mapping food is called");
+
+            return "food";
+    }
+
+    @GetMapping("/createFood")
+    public String createStudentForm(Model model) {
+
+            Food food = new Food();
+            model.addAttribute("food", food);
```

Comment:
TIP: You can also write it like this
`Food food = model.addAttribute("food", food);`
Then we save some line of code.